$MirOS: ports/emulators/qemu/patches/patch-target-i386_cpu_h,v 1.3 2005/11/15 12:25:09 tg Exp $
--- target-i386/cpu.h.orig	Sun Sep  4 17:11:09 2005
+++ target-i386/cpu.h	Tue Nov 15 11:37:45 2005
@@ -630,8 +630,7 @@ void cpu_x86_frstor(CPUX86State *s, uint
 /* you can call this signal handler from your SIGBUS and SIGSEGV
    signal handlers to inform the virtual CPU of exceptions. non zero
    is returned if the signal was handled by the virtual CPU.  */
-struct siginfo;
-int cpu_x86_signal_handler(int host_signum, struct siginfo *info, 
+int cpu_x86_signal_handler(int host_signum, qemu_siginfo *info, 
                            void *puc);
 void cpu_x86_set_a20(CPUX86State *env, int a20_state);
 
